test/auth-config.js

```diff
@@ -4,7 +4,7 @@ var Lab = require('lab'),
   test = lab.test,
   expect = require('code').expect;
 
-var auth = require('../lib/auth-config');
+var auth = require('../lib/auth-config')(function() {});
```

Comment:
This is a little odd. Why are we passing an empty function here? It might be better to make the argument optional.